src/gsi/m_berror_stats_reg.f90

```diff
@@ -368,6 +370,8 @@ subroutine berror_read_wgt_reg(msig,mlat,corz,corp,hwll,hwllp,vz,rlsig,varq,qopt
 !                         vertical length scale from 1/layer to 1/sigma
 !       2022-05-24 ESRL(H.Wang) - Add B for reginal FV3-CMAQ
 !                        (berror_fv3_cmaq_regional=.true.) . 
+!       2023-07-30 Zhao - added code for tuning the static BE of howv in hybrid
+!                         run (since ensemble of howv is not available yet).
```

Comment:
@GangZhao-NOAA  Would you please continue to  explain here how you will handle this situation when ensemble of howv is not  available when the hybrid analysis is done?

Reply:
@TingLei-NOAA ,
As you know, even no ensemble of howv, GSI still can run the hybrid analysis. For the B of howv, since there is no contribution from ensemble of howv, so the B of howv is the static B of howv multiplied by the weight of static BE. So in hybrid analysis, the analysis of HOWV is actually the pure variational analysis, with a reduced BE of howv (reduced by the weight).
If to make the analysis of HOWV in hybrid is similar to the analysis of howv in pure 3dvar, we could tune (actually increase) the static BE of howv according the weight of static BE used in hybrid analysis.
Thanks,

Reply:
@GangZhao-NOAA  Thanks for your explanation.  So theoretically ( without considering impacts on iteration processes),  the howv using 1/weight_B * "original static B(howv)" in the hybrid analysis would be the same as an independent 3dvar of only how  using the "original stati B(howv) " , right?   As mentioned above, one possibilities is the extra term in the obs related cost function could impact the iteration processes.  Did you run some comparision experiments to see if adding howv in the hybrid analysis cause differences in other variable analysis? 
And,  I think you could add this or more formal documentation on the treatment of howv in the hybrid analysis in the associated issue and some place in the codes, which would help a lot.

Reply:
@TingLei-NOAA 
Yes, analyzing howv obs does add new term in the cost function so that there are some differences in the analyses of other variables between analysis with vs without howv.  I did run the experiments and compared the results (you may find the breif report in issue #601 and this PR #614.

Reply:
@GangZhao-NOAA . Thanks for this explanation.  I would recommend this explanation (without those comparison details) be added in the codes about how howv is treated in the hybrid runs.

Reply:
@TingLei-NOAA
I will add a brief explanation about how howv is treated in the hybrid run. 
Thanks,